Dockerfile.tools

```diff
@@ -2,23 +2,17 @@ FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.6
 LABEL stage=build
 WORKDIR /build/
 
-# Build kubelet.exe
-RUN git clone --branch release-4.6 https://github.com/openshift/kubernetes.git
-WORKDIR /build/kubernetes/
-# TODO: Checking out commit before go1.15 became a requirement, revert checkout in https://issues.redhat.com/browse/WINC-460
-RUN git checkout f5121a6a6a02ddfafd2bfbf5201b092dc5097ab0
+RUN git clone --branch release-4.7 --single-branch --recurse-submodules https://github.com/openshift/windows-machine-config-operator.git
```

Comment:
Should we add an "inception" warning here given the WMCB repo is a sub-module? :-P

Reply:
Hm, I'm starting to think I we should probably only initialize the submodule repos that we actually need/build, otherwise we're cloning all of them each time.

Reply:
I like that approach